send/generic_send

```diff
@@ -11,10 +11,15 @@ TIMEOUT_KILL="60" # 60 sec to kill after timeout
 FACILITY_NAME=$1
 DESTINATION=$2
 DESTINATION_TYPE=$3
+KEY_PATH="`echo ~`/.ssh/id_rsa"
+
+if [ -f "/etc/perun/services/${SERVICE_NAME}/${SERVICE_NAME}.conf" ]; then
+	. "/etc/perun/services/${SERVICE_NAME}/${SERVICE_NAME}.conf"
+fi
 
 unset LC_TIME
 unset LANG
-TRANSPORT_COMMAND="ssh -o StrictHostKeyChecking=no -o GSSAPIAuthentication=no -o GSSAPIKeyExchange=no -o ConnectTimeout=5  -i `echo ~`/.ssh/id_rsa"
+TRANSPORT_COMMAND="ssh -o StrictHostKeyChecking=no -o GSSAPIAuthentication=no -o GSSAPIKeyExchange=no -o ConnectTimeout=5  -i $KEY_PATH"
```

Comment:
Let's improve it a bit, while we are making changes in this part of code. Please put $KEY_PATH into apostrophes, just in case someone want to use path containing space in there.